src/models/service/transactions/summary.rs

```diff
@@ -138,16 +138,29 @@ pub struct TransactionSummary {
     pub safe_app_info: Option<SafeAppInfo>,
 }
 
+#[derive(Serialize, Debug, PartialEq)]
+#[serde(tag = "type", rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum ExecutionInfo {
+    Multisig(MultisigExecutionInfo),
+    Module(ModuleExecutionInfo),
+}
```

Comment:
We would need to adjust the sample json in this file to contain ` "type" : "MULTISIG_EXECUTION_INFO"` If you are tight on time, we can open an issue to update the docs after we've finished all the api breaking changes.